src/docs/GetStarted/Developers/configureStylelint/index.mdx

```diff
@@ -61,6 +61,25 @@ Please checkout [Mozaic configuration](/GetStarted/Developers/Configuration/#sty
 
 </Highlight>
 
+### Add your own stylelint rules 
+
+It can appear that sometimes you need to use custom rules in stylelint configuration file for your project, you can add it this way in `stylelint.config.js`: 
```

Comment:
Sometimes you may need to customize Stylelint rules or add new ones.

To do this, you can expand your `stylelint.config.js` file this way: